.teamcity/Gradle_Promotion/buildTypes/bt61.xml

```diff
@@ -16,7 +16,7 @@
           <param name="teamcity.coverage.emma.instr.parameters" value="-ix -*Test*" />
           <param name="teamcity.coverage.idea.includePatterns" value="*" />
           <param name="teamcity.step.mode" value="default" />
-          <param name="ui.gradleRunner.additional.gradle.cmd.params" value="-PuseBuildReceipt &quot;-PgitUserName=Gradleware Git Bot&quot; &quot;-PgitUserEmail=gradlewaregitbot@gradleware.com&quot; -Igradle/buildScanInit.gradle" />
+          <param name="ui.gradleRunner.additional.gradle.cmd.params" value="-PuseBuildReceipt &quot;-PgitUserName=Gradleware Git Bot&quot; &quot;-PgitUserEmail=gradlewaregitbot@gradleware.com&quot; -Igradle/buildScanInit.gradle -Djava7Home=%linux.jdk.for.gradle.compile%" />
```

Comment:
We could actually use `-Pjava7Home=...` since we don't care about `buildSrc`. WDYT?

Reply:
We could yes. But what about we use a single way to set this property across the whole CI setup?

Reply:
That is fine with me, too.
Then the promote project would need to support setting a project property, too, since then I can setup my `gradle.properties` in my Gradle user home accordingly and run tasks on the promote project locally.